card/mod/02_basic_types/spec/set/type/pointer_spec.rb

```diff
@@ -28,19 +28,19 @@
 
   describe "drop_item" do
     it "remove the link" do
-      pointer = Card.new :name=>"tp", :type=>"pointer", :content=>"[[Jane]]\n[[John]]"
+      pointer = Card.new name: "tp", type: "pointer", content: "[[Jane]]\n[[John]]"
       pointer.drop_item "Jane"
       pointer.content.should == "[[John]]"
     end
 
     it "not fail on non-existent reference" do
-      pointer = Card.new :name=>"tp", :type=>"pointer", :content=>"[[Jane]]\n[[John]]"
+      pointer = Card.new name: "tp", type: "pointer", content: "[[Jane]]\n[[John]]"
```

Comment:
Line is too long. [83/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.